src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -1359,16 +1359,17 @@ public static LightningStrikeEvent createLightningStrike(Game game, WeatherVolum
      * Creates a new {@link WeatherChangeEvent}.
      *
      * @param game The game instance for this {@link GameEvent}
-     * @param weatherVolume The volume the weather changed in
+     * @param weatherUniverse The volume the weather changed in
      * @param initialWeather The previous weather
      * @param resultingWeather The weather to change to
      * @return A new instance of the event
      */
-    public static WeatherChangeEvent createWeatherChange(Game game, WeatherVolume weatherVolume, Weather initialWeather, Weather resultingWeather) {
+    public static WeatherChangeEvent createWeatherChange(Game game, WeatherUniverse weatherUniverse, Weather initialWeather,
+            Weather resultingWeather) {
         Map<String, Object> values = Maps.newHashMap();
         values.put("game", game);
         values.put("initialWeather", initialWeather);
-        values.put("weatherVolume", weatherVolume);
+        values.put("weatherVolume", weatherUniverse);
```

Comment:
Didn't notice it, this was done as an automatic refactor. Good catch, I'll fix it.